lib/swiftner/configuration.rb

```diff
@@ -3,6 +3,7 @@
 module Swiftner
   ## `Swiftner::Configuration`
   class Configuration
+    SUPPORTED_LANGUAGES = %w[en no].freeze
```

Comment:
Hardcoding this seems like a bad idea. Is there a way to fetch the current supported languages out of the swiftner backend?

Reply:
There doesn't seem to be a way to fetch the languages. Found this [constants.ts](https://github.com/Swiftner/swiftner_studio/blob/bc110e651712e921d21f1c32b00de7077e44c616/src/utils/constants.ts#L4). But they use whisper and they have a different [list](https://platform.openai.com/docs/guides/speech-to-text/supported-languages). I asked Mario earlier and he said to search for and use whisper's list maybe I should've done that?

Reply:
Lets leave it as is for now. Mario should consider having the available languages as part of a config object available in from the api